src/sentry/rules/processing/processor.py

```diff
@@ -337,16 +336,15 @@ def apply_rule(self, rule: Rule, status: GroupRuleStatus) -> None:
             return
 
         if slow_conditions or fast_conditions:
-            predicate_iter = (self.condition_matches(f, state, rule) for f in condition_list)
+            predicate_iter = (self.condition_matches(f, state, rule) for f in fast_conditions)
```

Comment:
`condition_list` was always set to `fast_conditions` so this just makes it easier to read